src/mococrw/openssl_wrap.h

```diff
@@ -48,6 +48,16 @@ struct SSLDeleter
     }
 };
 
+/*
+ * Deleter template which doesn't delete, e.g. to wrap const pointers
+ * but not delete them.
+ */
+struct noopDeleter
```

Comment:
Smart pointers are a  drop-in replacement that look like pointers but do memory management. If no memory management needs to be done, there is no need for smart pointers. This looks like over engineering to me. What's the point of having a smart pointer that does exactly nothing on delete?

Reply:
no practical point, the only argument was keeping it coherent with the rest of pointers returned by the library (the comment in the file suggests "Wrap all the pointer-types returned by openssl.")

Reply:
changed to using the direct OpenSSL type